src/Resources/views/CRUD/display_date.html.twig

```diff
@@ -0,0 +1,18 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{%- spaceless %}
+    {%- if value is empty -%}
+        &nbsp;
+    {%- else -%}
+        <time datetime="{{ value|date('Y-m-d', 'UTC') }}" title="{{ value|date('Y-m-d', 'UTC') }}">{{ value|date(field_description.options.format|default('F j, Y'), field_description.options.timezone|default(null)) }}</time>
```

Comment:
Please use linebreaks here and below:
```suggestion
        <time datetime="{{ value|date('Y-m-d', 'UTC') }}"
              title="{{ value|date('Y-m-d', 'UTC') }}">
	   {{- value|date(
		   field_description.options.format|default('F j, Y'),
		   field_description.options.timezone|default(null)
	   ) -}}
        </time>
```

Reply:
This is more readable to you? oO

Reply:
If I don't have to touch my mouse to read it then it is obviously more readable. I'm open to other ways to put the linebreaks though, but ultimately, there should be some linebreaks in there.

Reply:
Also, I just fixed an indention issue, there seemed to be mixed tabs and spaces :/

Reply:
I'm not again line breaks in general. With this solution you could miss that there is a `,`.

Maybe you could introduce an options variable to make this a oneliner.
```twig
{% set options = field_description.options %}
```

Reply:
I put back the html attributes on one line, because it's still under 120 chars, although it is over 80. Regarding the extra variable, not a huge fan… with the solution above, it is easy to see that `data` takes 2 arguments and what they are. Having `{{- value|date(options.format|default('F j, Y'), options.timezone|default(null)) -}} would hardly be more readable, and an extra assignment is more cognitive load IMO.